server/migration/1745268105339-mercury_wasm.ts

```diff
@@ -0,0 +1,18 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class MercuryWasm1745268105339 implements MigrationInterface {
+    name = 'MercuryWasm1745268105339'
```

Comment:
what do these numbers mean 😂

Reply:
I believe they're timestamps – the ORM generates them when you generate a migration from changing an entity

Reply:
Also, they're fun! (? maybe?)